parser/parser_utils.py

```diff
@@ -258,6 +227,30 @@ def visit_delete(self, node, context):
                     self.process(node.where, context)
                 return None
 
+            def visit_between_predicate(self, node, context):
+                if isinstance(node.value, QualifiedNameReference):
+                    self.add_filter_column_with_qualified_name_reference(node.value, 'between')
+                return None
+
+            def add_filter_column_with_qualified_name_reference(self, qualified_name_reference: QualifiedNameReference,
```

Comment:
The subsequent processing part needs to increase the processing of value and context, such as the transfer of fields when multi-table association